kubernetes/client/api_client.py

```diff
@@ -193,7 +193,7 @@ def sanitize_for_serialization(self, obj):
                     for sub_obj in obj]
         elif isinstance(obj, tuple):
             return tuple(self.sanitize_for_serialization(sub_obj)
-                for sub_obj in obj)
+                         for sub_obj in obj)
```

Comment:
scripts/update-pep8.sh does not update this file. If this is the right style we should have, we need to be able to automate it there.